uxarray/grid.py

```diff
@@ -540,6 +542,73 @@ def _build_edge_node_connectivity(self, repopulate=False):
         setattr(self, "Mesh2_edge_nodes", self.ds['Mesh2_edge_nodes'])
         setattr(self, "nMesh2_edge", edge_nodes_unique.shape[0])
 
+    def _build_node_faces_connectivity(self):
```

Comment:
Should be `node_face` instead of `node_faces` throughout

```suggestion
    def _build_node_face_connectivity(self):
```

Reply:
According to our naming conventions, it should be `node_faces` since it records the neighboring faces of a node. Same as our `edge_nodes_connectivity`, `face_nodes_connectivity`

Reply:
Got it, let's keep it the way it is then. Even after looking at the UGRID document 1000x times I still mess up these little things :)